crystal_diffusion/train_mtp.py

```diff
@@ -0,0 +1,202 @@
+"""Script to train and evaluate a MTP.
+
+Running the main() runs a debugging example. Entry points are train_mtp and evaluate_mtp.
+"""
+import os
+from typing import Any, Dict, List, Tuple
+
+import numpy as np
+import pandas as pd
+import yaml
+from pymatgen.core import Structure
+from sklearn.metrics import mean_absolute_error
+
+from crystal_diffusion.models.mtp import MTPWithMLIP3
+
+MLIP_PATH = os.path.join(os.getcwd(), "mlip-3")
+SAVE_DIR = os.path.join(os.getcwd(), "debug_mlip3")  # for demo only
+
+# TODO list of yaml files should come from an external call
+# yaml dump file
+lammps_yaml = ['lammps_scripts/Si/si-custom/dump.si-300-1.yaml']
+# yaml thermodynamic variables
+lammps_thermo_yaml = ['lammps_scripts/Si/si-custom/thermo_log.yaml']
+# note that the YAML output does not contain the map from index to atomic species
+# this will have to be taken from elsewhere
+# use a manual map for now
+atom_dict = {1: 'Si'}
+
+
+def extract_structure_and_forces_from_file(filename: str, atom_dict: Dict[int, Any]) -> \
```

Comment:
There is no unit test for this method.

Reply:
I don't think it does. The point particle implication here is not clear.

<img width="722" alt="image" src="https://github.com/mila-iqia/diffusion_for_multi_scale_molecular_dynamics/assets/35506852/e1c4ed04-dc05-4a28-b107-a2767cc224e4">

Reply:
added unit tests